app/src/main/res/values/strings.xml

```diff
@@ -454,7 +454,7 @@
     <string name="fireproofWebsiteSnackbarConfirmation">&lt;b>%s&lt;/b> is now fireproof! Visit Settings to learn more.</string>
     <string name="fireproofWebsiteSnackbarAction">Undo</string>
     <string name="fireproofWebsiteDeleteConfirmMessage">Are you sure you want to delete &lt;b>%s&lt;/b>?</string>
-    <string name="fireproofWebsiteEmptyListHint">No websites fireproofed yet</string>
+    <string name="fireproofWebsiteEmptyListHint">No websites Fireproofed yet</string>
```

Comment:
I don't think so. It's just about capitalizing the word in English in the same way we capitalize Fire Button. But in other languages, because we rely on translations, we don't have a clear keyword/concept to capitalize there.